firebase/public

```diff
@@ -0,0 +1 @@
+../build
```

Comment:
This is a symlink from `firebase/public` to `../build` because Firebase won't let me point to a parent directory of firebase.json.  

It's a bit of a hack.  We could move things around to avoid the symlink, but it seems to work for now. 🤷